<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RecipeCard</class>
 <widget class="QWidget" name="RecipeCard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>200</width>
    <height>150</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>200</width>
    <height>150</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>200</width>
    <height>150</height>
   </size>
  </property>
  <property name="styleSheet">
   <string notr="true">RecipeCard&gt;QFrame {
    border: 1px solid #ccc; /* Grubość i kolor obramowania */
    border-radius: 5px; /* Zaokrąglone rogi */
}

/* Styl dla całego widgetu RecipeCard */
RecipeCard {
    background-color: #f0f0f0; /* Kolor tła */
    padding: 10px; /* Wewnętrzne odstępy */
}

/* Pseudoelement outline dla kontenera RecipeCard */
RecipeCard::outline {
    outline: 1px solid #ccc; /* Grubość i kolor obramowania */
    border-radius: 5px; /* Zaokrąglone rogi */
}

/* Styl dla obrazu na górze karty */
RecipeCard #imageLabel {

}

/* Styl dla nazwy przepisu */
RecipeCard #titleLabel {
    font-weight: bold; /* Pogrubiona czcionka */
    text-align: left; /* Wyrównanie do lewej */
}

/* Styl dla opisu przepisu */
RecipeCard #descriptionLabel {
    color: #666; /* Kolor tekstu */
    text-align: left; /* Wyrównanie do lewej */
}
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="frame">
     <layout class="QVBoxLayout" name="frameLayout">
      <item>
       <widget class="QLabel" name="imageLabel">
        <property name="text">
         <string/>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="titleLabel">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Przepis</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="descriptionLabel">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Opis przepisu</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
